### PR TITLE
feat: add footer in layout

### PR DESCRIPTION
**Pull request by divyanshugarg36**
- remove react imports from unrequired places
- add interface for constant data

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            linkedin: LinkedInIcon,
+            twitter: TwitterIcon,
+            youtube: YoutubeIcon,
+          }[item.title as 'facebook' | 'linkedin' | 'twitter' | 'youtube'];
+          return (
+            <Link key={item.value} href={item.value}><a><Icon /></a></Link>
+          );
+        })}
+      </div>
+      <Link href="https://bestpractices.coreinfrastructure.org/projects/66">
```
Comment (theamanjs) [inline src/components/Footer.tsx]: You can put the links in a constant file and map them from there.

**Review by theamanjs — CHANGES_REQUESTED** [review]

File: src/components/Footer.tsx (hunk 1 above)
Comment (divyanshugarg36) [reply src/components/Footer.tsx]: This one is fixed and pushed

Comment (theamanjs) [reply src/components/Footer.tsx]: Can you link the changes here? @divyanshugarg36

Comment (divyanshugarg36) [reply src/components/Footer.tsx]: This is the commit showing these changes
https://github.com/divyanshugarg36/brl-cad-web/pull/3/commits/28b71fd6805a84b08c247b97acf06a0160ee441e

**Review by theamanjs — APPROVED** [review]

**Pull request merged**